sorc/build.upp

```diff
@@ -13,6 +13,8 @@ rm -rf ${HOMErrfs}/sorc/UPP/tests/build
 rm -rf ${HOMErrfs}/sorc/UPP/tests/install
 rm -rf ${EXEC}
 
+cp ${HOMErrfs}/sorc/_workaround_/INITPOST_MPAS.F  ${HOMErrfs}/sorc/UPP/sorc/ncep_post.fd/INITPOST_MPAS.F
```

Comment:
Could you put `INITPOST_MPAS.F` inside `_workaround_/UPP`? Thanks!

Reply:
Done. Thanks!